dask_image/dispatch/_dispatch_ndinterp.py

```diff
@@ -30,6 +30,44 @@ def cupy_affine_transform(*args, **kwargs):
         return cupyx.scipy.ndimage.affine_transform
 
 
+dispatch_spline_filter = Dispatcher(name="dispatch_spline_filter")
+
```

Comment:
Would you mind running flake8 on the files for this PR? There's a few formatting changes it'll suggest (like here, it will ask you for two blank lines before a function definition).

Reply:
done!